code/game/gamemodes/shadowling/shadowling_abilities.dm

```diff
@@ -103,6 +103,10 @@
 	for(var/obj/item/F in H)
 		if(F.light_range > 0)
 			extinguishItem(F)
+		if(ishuman(H))
```

Comment:
This should be tabbed one level down, or this is going to be unnecessarily checked for every item in the mob.